conf/app.ini

```diff
@@ -62,6 +62,11 @@ handlers = console
 qualname = sentry.errors
 propagate = 0
 
+[logger_ws4py]
```

Comment:
Testing this locally by applying the change here to `conf/development-app.ini` I get a message about a missing required "handlers" argument.